spring-cql/src/test/java/org/springframework/cassandra/test/integration/config/xml/TestLatencyTracker.java

```diff
@@ -18,13 +18,15 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.datastax.driver.core.Cluster;
 import com.datastax.driver.core.Host;
 import com.datastax.driver.core.LatencyTracker;
 import com.datastax.driver.core.Statement;
 
 /**
  * @author David Webb
  * @author Oliver Gierke
+ * @author Antoine Toulme
  */
 public class TestLatencyTracker implements LatencyTracker {
```

Comment:
Maybe something that could have been "mocked" instead, given this test class implements the `LatencyTracker` interface from DataStax.  Even if used in XML, it is still possible to mock.